src/h5web/vis-packs/core/complex/MappedComplexVis.tsx

```diff
@@ -83,11 +87,19 @@ function MappedPhaseMapVis(props: Props) {
 
   return (
     <HeatmapVis
-      dataArray={phaseArray}
-      domain={safePhaseDomain}
-      alphaArray={amplitudeArray}
-      alphaDomain={amplitudeDomain}
-      title={title}
+      dataArray={dataArray}
+      domain={safeDomain}
+      alphaArray={
+        visType === ComplexVisType.PhaseAndAmplitude
+          ? amplitudeArray
+          : undefined
+      }
+      alphaDomain={
+        visType === ComplexVisType.PhaseAndAmplitude
```

Comment:
Could you maybe renamed `PhaseAndAmplitude` to just `PhaseAmplitude`? The `And` seems unnecessary.